raiden/transfer/state.py

```diff
@@ -603,15 +604,15 @@ def get_address_metadata(
     return None
 
 
-def fetch_address_metadata(
-    address: Address, route_states: Optional[List[RouteState]] = None
-) -> AddressMetadata:
+def fetch_address_metadata(address: Address) -> AddressMetadata:
     from raiden.network.pathfinding import query_address_metadata
     from raiden.raiden_service import RaidenService
 
-    metadata = get_address_metadata(address, route_states) if route_states else {}
-    metadata = metadata or {}
+    metadata = {}
     pfs_config = RaidenService.pfs_config()
-    if not metadata and pfs_config is not None:
-        metadata = query_address_metadata(pfs_config, address)
+    if pfs_config is not None:
+        try:
+            metadata = query_address_metadata(pfs_config, address)
+        except ServiceRequestFailed:
+            pass
```

Comment:
Maybe add some logging here to let the user know that it was not able to fetch metadata. It will result in not being able to send a message.

Reply:
I added the log to the raiden_event_handler, this function has now been removed ✅